jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/wizard/SchemaSelectionPanelTest.kt

```diff
@@ -223,13 +190,53 @@ class SchemaSelectionPanelTest {
         assertThat(schemaTemplateParameters?.templateExtraContext?.schemaPackageHierarchy).isEqualTo(schemaPackageHierarchy)
         assertThat(schemaTemplateParameters?.templateExtraContext?.source).isEqualTo(DEFAULT_EVENT_SOURCE)
         assertThat(schemaTemplateParameters?.templateExtraContext?.detailType).isEqualTo(DEFAULT_EVENT_DETAIL_TYPE)
-        assertThat(schemaTemplateParameters?.templateExtraContext?.userAgent).isEqualTo(AWSToolkitUserAgent)
+        assertThat(schemaTemplateParameters?.templateExtraContext?.userAgent).isEqualTo(AWS_TOOLKIT_USER_AGENT)
+    }
+
+    private fun createSelector(): SchemaResourceSelector {
+        val selector = runInEdtAndGet {
+            initMockResourceCache()
+
+            val schemaSelectionSelector = SchemaResourceSelector()
+            schemaSelectionSelector.awsConnection = connectionManager.settingsManager.connectionSettings()
+            schemaSelectionSelector.reload()
+
+            schemaSelectionSelector
+        }
+
+        selector.schemasSelector.waitToLoad()
+
+        return selector
     }
 
-    private fun load() {
-        initMockResourceCache()
-        schemaSelectionPanel.awsConnection = connectionManager.settingsManager.connectionSettings()
-        schemaSelectionPanel.reload()
-        schemaSelectionPanel.schemasSelector.waitToLoad()
+    private companion object {
```

Comment:
Why are these moved to a companion object?

Reply:
1. Naming convnetion is all uppercase is for static values so  they should be in companion
2. Constant vals should be in static fields, not per instance